backtranslation.py

```diff
@@ -0,0 +1,216 @@
+idiomas = ["en", "zh", "ar", "ru", "fr", "de", "es", "pt", "it", "ja", "ko",
+           "el", "nl", "hi", "tr", "ms", "th", "vi", "id", "he", "pl", "mn",
+           "cs", "hu", "et", "bg", "da", "fi", "ro", "sv", "sl", "fa", "bs",
+           "sr", "fj", "tl", "ht", "ca", "hr", "lv", "lt", "ur", "uk", "cy",
+           "ty", "to", "sw", "sm", "sk", "af", "no", "bn", "mg", "mt", "otq",
+           "tlh", "gu", "ta", "te", "pa", "am", "az", "ba", "be", "ceb", "cv",
+           "eo", "eu", "ga", "emj", "zh-CHS", "zh-CHT", "wyw", "yue", "mww"]
+
+def select_backtranslation (original, idioma, tradutor):
+    result = ""
+    try:
+        if (tradutor == 0):
+            trad = ts.google(original, if_use_cn_host=False, from_language='pt', to_language=idioma)
+            result = ts.google(trad, if_use_cn_host=False, from_language=idioma, to_language='pt')
+        elif (tradutor == 1):
+            trad = ts.bing(original, if_use_cn_host=False, from_language='pt', to_language=idioma)
+            result = ts.bing(trad, if_use_cn_host=False, from_language=idioma, to_language='pt')
+        elif (tradutor == 2):
+            trad = ts.sogou(original, is_detail_result=False, from_language='pt', to_language=idioma)
+            result = ts.sogou(trad, is_detail_result=False, from_language=idioma, to_language='pt')
+        elif (tradutor == 3):
+            trad = ts.translate_html(original, translator=ts.google, from_language='pt', to_language=idioma, n_jobs=1, timeout=20.0)
+            result = ts.translate_html(trad, translator=ts.google, from_language=idioma, to_language='pt', n_jobs=1, timeout=20.0)
+        elif (tradutor == 4):
+            trad = ts.translate_html(original, translator=ts.bing, from_language='pt', to_language=idioma, n_jobs=1, timeout=20.0)
+            result = ts.translate_html(trad, translator=ts.bing, from_language=idioma, to_language='pt', n_jobs=1, timeout=20.0)
+        elif (tradutor == 5):
+            trad = ts.translate_html(original, translator=ts.alibaba, from_language='pt', to_language=idioma, n_jobs=1, timeout=20.0)
+            result = ts.translate_html(trad, translator=ts.alibaba, from_language=idioma, to_language='pt', n_jobs=1, timeout=20.0)
+        elif (tradutor == 6):
+            trad = ts.translate_html(original, translator=ts.sogou, from_language='pt', to_language=idioma, n_jobs=1, timeout=20.0)
+            result = ts.translate_html(trad, translator=ts.sogou, from_language=idioma, to_language='pt', n_jobs=1, timeout=20.0)
+        elif (tradutor == 7):
+            trad = ts.translate_html(original, translator=ts.deepl, from_language='pt', to_language=idioma, n_jobs=1, timeout=20.0)
+            result = ts.translate_html(trad, translator=ts.deepl, from_language=idioma, to_language='pt', n_jobs=1, timeout=20.0)
+    except: result = ""
+    return result
+
+def parallel_translation (original, rate, idioma, tradutor, currtags, traducoes):
```

Comment:
In function declaration the Python style guide says to not put spaces between name and parenthesis, it should be in this case parallel_translation(...).

Reply:
Solved.